Cargo.toml

```diff
@@ -1,11 +1,5 @@
 [workspace]
-members = [
-    ".",
-    "cli",
-    "derive",
-    "discoverer",
-    "generator",
-]
+members = [".", "cli", "derive", "discoverer", "generator"]
```

Comment:
May I ask what tool you used to format toml?

Reply:
> Welcome to the community!
> 
> Does this PR depends on the new features offered in `async-graphql` 5.0? If not, may be we can separate this feature and the upgrade?

As far as I know the PR does not depend on async-graphql 5.0. It is a dependency for my project using seaography. It can be seperated but I would like to have a bump to async-graphql 5.0 in the official repo.

Reply:
> May I ask what tool you used to format toml?

`rustfmt` nightly

Reply:
> May I ask what tool you used to format toml?

https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml

Reply:
Ah okay, it looks nice indeed. But we kind of want to have a cli we can use on CI so as to maintain standard.